src/display/TopBar/TopBar.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import { Meta } from "@storybook/react";
+import React from "react";
+import styled from "styled-components";
+import { heading1, sizing, theme } from "src/guidelines/theme";
+import TopBar, { TopBarProps } from "./TopBar";
+
+const Section = styled.div`
+  border: 1px solid ${theme.color.primary100};
+  padding: ${sizing(16)};
+  border-radius: ${theme.borderRadius.default};
+  text-align: center;
+  color: ${theme.color.primary700};
+  background-color: ${theme.color.primary100};
+`;
+
+const meta: Meta<typeof TopBar> = {
+  component: TopBar,
+  title: "Display/TopBar",
+};
+
+export default meta;
+
+const Title = styled.h1`
+  ${heading1};
+  margin: ${sizing(24)} 0 ${sizing(16)};
+`;
+
+export const Demo = {
+  args: {
+    leftFlex: 1,
+    centerFlex: 1,
+    rightFlex: 1,
+    gap: "xxlarge",
+    spacing: "flex-start",
+    leftSection: <Section>LEFT</Section>,
+    centerSection: <Section>CENTER</Section>,
+    rightSection: <Section>RIGHT</Section>,
+    "aria-label": "TopBar Demo",
+  },
+  argTypes: {
+    gap: {
+      control: "select",
+      options: ["small", "medium", "large", "xlarge", "xxlarge"],
+    },
+    spacing: {
+      control: "select",
+      options: ["start", "center", "end", "around", "evenly", "between"],
+    },
+    leftFlex: {
+      control: { type: "number", min: 0, max: 3 },
+    },
+    centerFlex: {
+      control: { type: "number", min: 0, max: 3 },
+    },
+    rightFlex: {
+      control: { type: "number", min: 0, max: 3 },
+    },
+    "aria-label": {
```

Comment:
I thnik this one is useless, in args aria-label is a string

Reply:
removed